### PR TITLE
Add color for the IV scanner

### PR DESCRIPTION
**Pull request by LeoDoby**
add a new color SUMMARY_GREEN in text.js, add the green color if the iv of the pokemon is better than the iv on the starter of that pokemon

### CODE REVIEW

**Comment by EmoUsedHM01** [comment]
This looks amazing btw!
https://i.imgur.com/Pcp3pmP.png

File: src/ui/battle-message-ui-handler.ts [hunk 1]
```diff
@@ -221,18 +222,38 @@ export default class BattleMessageUiHandler extends MessageUiHandler {
...
-    if (value > 10)
-      return 'Pretty Good';
-    if (value)
-      return 'Decent';
-    return 'No Good';
+  getIvDescriptor(value: integer, typeIv: integer): string {
+    let starterIvs: number[] = this.scene.gameData.dexData[this.scene.getEnemyPokemon().species.speciesId].ivs;
+    let uiTheme = (this.scene as BattleScene).uiTheme; // Assuming uiTheme is accessible
+
+    // Function to wrap text in color based on comparison
```
[the remark this answers is not in the document]
Comment (LeoDoby) [reply src/ui/battle-message-ui-handler.ts:229]: so is something like that ok ?

[the remark this answers is not in the document]
Comment (molker) [reply src/ui/battle-message-ui-handler.ts:229]: yeah that looks better, need to test this but thanks for cleaning that up

[the remark this answers is not in the document]
Comment (EmoUsedHM01) [reply src/ui/battle-message-ui-handler.ts:229]: I tested and it worked perfectly fine, but just give it a try, honestly unplayable without this now

**Review by molker — APPROVED** [review]
lgtm, thanks for making the changes

**Pull request merged**